tests/main/snapd-snap/task.yaml

```diff
@@ -80,7 +80,11 @@ prepare: |
         modprobe fuse
 
         echo "Install lxd"
-        snap install lxd --channel="$LXD_SNAP_CHANNEL"
+        if os.query is-pc-i386; then
+            snap install lxd --channel="3.0/$LXD_SNAP_CHANNEL"
+        else
+            snap install lxd --channel="$LXD_SNAP_CHANNEL"
+        fi
```

Comment:
@sergiocazzolato probably need to reach out to the lxd team, not sure what to do about that

Reply:
> spread.yaml because this variables are defined in the host where spread is executed and os.query is not in path yes

spread.yaml supports both host & target env variables. those that are prefixed with HOST are executed where the spread is running; whereas without HOST it is executed in the target environment, and correctly sets 3.0/stable channel on the 14.04 runners.

re snapcraft not able to launch things with lxd 3.0 => it is possible that snapcraft no longer supports lxd 3.0 series, in that case we can't anymore build i386 snaps using snapcraft and lxd provider =(